README.md

```diff
@@ -5,8 +5,10 @@ multiple crates, each benchmark has one "crate of interest" which is the one
 whose compilation time is measured.
 
 Each benchmark has a makefile with the following targets.
-* `all`: builds the entire benchmark. The `CARGO_RUSTC_OPTS` environment
-  variable can be specified to pass extra arguments to rustc invocations.
+* `all`: builds the entire benchmark. The following environment variables can
+  be used to influence how it is built.
+  * `CARGO_OPTS`: extra arguments to the `cargo` invocation.
+  * `CARGO_RUSTC_OPTS`: extra arguments to `rustc` invocations.
```

Comment:
Is this referring to `cargo rustc` invocations? Otherwise it's probably just RUSTC_OPTS, right?